docs/guides/bridge.md

```diff
@@ -1,16 +1,14 @@
-# kBTC/interBTC Bridge Guide (Testnet)
+# kBTC/interBTC Bridge Guide
 
-?> Warning: These guides are for Testnet only! Kintsugi and Interlay bridge guides are WIP. The process is the same but specific naming might have changed. 
+## Issue
 
-## Issue (Testnet)
-
-interBTC allows you to receive a representation of BTC to be used any way you see fit in the Polkadot ecosystem.
+kBTC or interBTC allows you to receive a representation of BTC to be used any way you see fit in the Polkadot ecosystem.
 To get you started, follow this guide.
 
 At the end of this guide you will have:
 
 - [x] Locked BTC with a collateralized Vault
```

Comment:
```suggestion
- [x] Requested to mint kBTC/interBTC on the Kintsugi/Interlay parachain
- [x] Sent BTC to a Vault on the Bitcoin blockchain
- [x] Received kBTC/interBTC on the Kintsugi/Interlay parachain
```

And can you add a link to the section where the user is doing these things like in: https://interbtc-docs-git-theo-update-bridge-interlay.vercel.app/#/guides/governance?id=governance

Same on redeem